hosting/client/components/PDFView/PDFView.vue

```diff
@@ -33,6 +36,15 @@ export default {
     },
     name() {
       return this.$store.state.name;
+    },
+    type() {
```

Comment:
Um die Nummer und den Namen beim wechsel auf die Einzellegende zu löschen, könnte man den Setter anpassen:

`type: {
  get() {
    return this.$store.state.type;
  },
  set(value) {
    if (value == 'single') {
      this.$store.commit("setSeries", '');
      this.$store.commit("setNumber", '');
    }
    this.$store.commit("setType", value);
  }
},`

Alternativ müsste man die Legendenkarte anpacken, aber dann würde die alte Fehlerhafte Information bleiben.